src/python/gem5/components/processors/abstract_generator.py

```diff
@@ -35,6 +35,20 @@
 from typing import List
 
 
+def _devide_memory_ranges(min_addr, max_addr, num_partitions):
```

Comment:
Two TODOs here:

- Rename `_divide_memory_ranges`
- Add typing (ints for the parameter and the return type should be `List[int]` if I'm reading this correctly.

Reply:
Done.